docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -60,7 +61,7 @@ If you cannot install GraalVM, you can use a multi-stage Docker build to run Mav
 Version {graalvm-version} is required. Using the community edition is enough.
 
 1. Install GraalVM if you haven't already. You have a few options for this:
-** Use platform-specific install tools like https://github.com/graalvm/homebrew-tap[homebrew] or https://sdkman.io/jdks#Oracle[sdkman].
+** Use platform-specific install tools like https://github.com/graalvm/homebrew-tap[homebrew] or https://sdkman.io/jdks#Oracle[sdkman] or https://github.com/ScoopInstaller/Java[scoop].
```

Comment:
For Windows I would mention Chocolatey tool, not just scoop. 
Chocolatey is very popular on Windows platform - 6.3k stars on Chocolatey GH vs. 358 stars for scoop.

https://chocolatey.org/packages/graalvm

Reply:
scoop has the advantage to not require admin so of such its a simpler model.

Reply:
The main scoop github repo actually has [9.9k](https://github.com/lukesampson/scoop) stars. It's also much easier to contribute to (in fact I just had the PR to export GRAALVM_HOME accepted last night), but I can include choco as well.

Reply:
ah, I looked at https://github.com/ScoopInstaller

not require admin - that's a strong argument for `scoop`

Consider my comment about `choco` inclusion as optional